source/tutorials/qiime2-for-experienced-microbiome-researchers.rst

```diff
@@ -0,0 +1,296 @@
+
+Assigning taxonomy
+~~~~~~~~~~~~~~~~~~
+
+Assigning taxonomy to ASV or OTU representative sequences is covered in the :ref:`taxonomy classification
+tutorial <Taxonomy>`.
+All taxonomy assignment methods are in the :doc:`feature-classifier plugin <../plugins/available/feature-classifier/index>`.
+
+There are two main approaches for assigning taxonomy, each with multiple methods available.
+
+The first involves aligning reads to reference databases directly:
+
+- :doc:`classify-consensus-blast <../plugins/available/feature-classifier/classify-consensus-blast/>`: BLAST+ local alignment
+- :doc:`classify-consensus-vsearch <../plugins/available/feature-classifier/classify-consensus-vsearch/>`:  VSEARCH global alignment
+
+Both use the *consensus* approach of taxonomy assignment, which you can learn more about in the overview and tweak with the ``maxaccepts``, ``perc-identity``, and ``min-consensus`` parameters.
+
+The second approach uses machine learning classifiers to assign likely taxonomies to reads:
+
+- :doc:`fit-classifier-sklearn <../plugins/available/feature-classifier/fit-classifier-sklearn/>`
+- :doc:`fit-classifier-naive-bayes <../plugins/available/feature-classifier/fit-classifier-naive-bayes/>`
+
+These two functions differ in the type of machine learning model that they use.
+(To get more details on the specific models implemented, check out `the plugin's associated paper <https://doi.org/10.1186/s40168-018-0470-z>`__.)
+These methods need a pre-trained model as one of the inputs: you can either download one of the pre-trained taxonomy classifiers from the :doc:`data resources page <../data-resources/>`, or train one yourself (following the steps outlined in the :doc:`feature classifier tutorial <feature-classifier>`).
+
+Analyze feature table and gain insight
+--------------------------------------
+
+At this point, you should be ready to analyze your feature table to answer your scientific questions!
+QIIME 2 offers multiple built-in functions to analyze your data, and you can also `export <Export the data>`_ it to do downstream analyses in your preferred coding language.
+
+Some general things you can do with QIIME 2 are:
+
+-  **Look at the data:** QIIME 2 has some a nice :doc:`taxa barplot visualizer <../plugins/available/taxa/barplot/>` to make visually exploring your data easy. You can also visualize your data on a PCoA plot with the :doc:`emperor <../plugins/available/emperor/plot/>` plugin (after calculating beta diversity between samples).
+-  **Build a phylogenetic tree:** QIIME 2 has a :doc:`phylogeny <../plugins/available/phylogeny/index>` plugin with different tree-building methods.
+-  **Calculate alpha diversity of your samples:** the :doc:`diversity plugin <../plugins/available/diversity/index>` has many `alpha diversity metrics <https://forum.qiime2.org/t/alpha-and-beta-diversity-explanations-and-commands/2282>`__ available through the ``alpha`` and ``alpha-phylogenetic`` methods.
```

Comment:
the `core-metrics*` pipelines are somewhat more user friendly and streamlined (since e.g., beta is followed by pcoa is followed by emperor). Could link to the flowchart in the overview tutorial if it's useful here.